spec/models/comment_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it 'is valid with content and a post_id' do
+    user = User.create(name: 'John', posts_counter: 0)
+    post = Post.create(title: 'Test Post', comments_counter: 0, likes_counter: 0, author: user)
+    comment = Comment.new(text: 'Test comment', post:, author: user)
+    expect(comment).to be_valid
+  end
+
+  it 'is invalid without content' do
+    user = User.create(name: 'John', posts_counter: 0)
+    post = Post.create(title: 'Test Post', comments_counter: 0, likes_counter: 0, author: user)
+    comment = Comment.new(post:, author: user)
+    expect(comment).to_not be_valid
+  end
+
+  it 'is invalid with too long content' do
+    user = User.create(name: 'John', posts_counter: 0)
+    post = Post.create(title: 'Test Post', comments_counter: 0, likes_counter: 0, author: user)
+    comment = Comment.new(text: 'A' * 1001, post:, author: user)
+    expect(comment).to_not be_valid
+  end
+
+  it 'is invalid without a post_id' do
+    user = User.create(name: 'John', posts_counter: 0)
+    comment = Comment.new(text: 'Test comment', author: user)
+    expect(comment).to_not be_valid
+  end
```

Comment:
- [ ] kindly add a test to prove the private method `update_comments_counter` works, you can do that by calling the `after_save method`